ovos_wolfram_alpha_solver/__init__.py

```
@@ -164,24 +158,26 @@ def make_speakable(summary: str):
         summary = " ".join(words)
         return rm_parentheses(summary)
 
-    # data api
-    def get_data(self, query: str,
-                 lang: Optional[str] = None,
-                 units: Optional[str] = None):
+    def query(self, query: str, lang: Optional[str] = None, k: int = 3) -> List[Tuple[str, float]]:
```

Ruff (0.15.0)

[warning] Unused method argument: `k`

(ARG002)